### PR TITLE
Initial commit of respository tools and verification

### PR DESCRIPTION
**Pull request by btcdrak**

### CODE REVIEW

File: contrib/devtools/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+===============
+
+A small script to automate merging pull-requests securely and sign them with GPG.
+
+For example:
+
+  ./github-merge.py 3077
+
+(in any git repository) will help you merge pull request #3077 for the
+bitcoin/bitcoin repository.
```
Comment (MarcoFalke) [inline contrib/devtools/README.md]: wrong repo

File: contrib/devtools/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+This means that there are no potential race conditions (where a
+pullreq gets updated while you're reviewing it, but before you click
+merge), and when using GPG signatures, that even a compromised github
+couldn't mess with the sources.
+
+Setup
+---------
+Configuring the github-merge tool for the bitcoin repository is done in the following way:
+
+    git config githubmerge.repository bitcoin/bitcoin
```
Comment (MarcoFalke) [inline contrib/devtools/README.md]: wrong config

**Comment by MarcoFalke** [comment]
Concept ACK

File: contrib/verify-commits/trusted-keys [hunk 3]
```diff
@@ -0,0 +1 @@
+1E7DD3547F244C757223FAC8FC6434FC3A31E956
```
Comment (btcdrak) [inline contrib/verify-commits/trusted-keys]: Need to add more trusted keys

**Comment by dexX7** [comment]
Are you going to use the verify commit script with Travis CI?

File: contrib/verify-commits/trusted-keys (hunk 3 above)
Comment (btcdrak) [inline contrib/verify-commits/trusted-keys]: @harding @laanwj @jonasschnelli what is your GPG signing key fingerprint?

**Comment by btcdrak** [comment]
@dexX7 I dont think that works, since the post important part if for the merge maintainer to sign and that is done after CI runs and pushed manually using the merge script.

File: contrib/verify-commits/trusted-keys (hunk 3 above)
Comment (jonasschnelli) [inline contrib/verify-commits/trusted-keys]: my GPG key fingerprint is 32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC

-----BEGIN PGP MESSAGE-----
Version: GnuPG v1

owEBeQKG/ZANAwACASnUvLZBb1PsActJYgBXvWG4bXkgR1BHIGtleSBmaW5nZXJw
cmludCBpcyAzMkVFNUM0QzNGQTE1Q0NBREI0NkFCRTUyOUQ0QkNCNjQxNkY1M0VD
CokCHAQAAQIABgUCV71huAAKCRAp1Ly2QW9T7P7sD/oDyR6DmdwqPcLSaxJvwvGZ
2CjQeO88IVafLvPG193wvhFNbg62cDk6f8jvAaqJvedIMLjNL1uWAVlwcINPmccJ
8VhjVX5RjpLOHJszlW2FEryY7Pax47htofmyq4avMxdJJfKv/chCwJJ/oxzFQ7TJ
UIKtttaTy9sbb0DPYOWqJyAhyErU1eBP3oOpixr1+H+kRREJPjsTfDEoBiNsgdiK
xnziB9i0+NVYAjUWHSbusrkAIQz6a7CSspChslsFALTxtWIgWDcrZXjYeVGNShPG
wSBYTpyr7XTTSB5Q+eTTO2cbJP3Rcv4RKeFy5aUsdDiMga4JG8mlvLq2rGGvzAcd
qo8IZ+Rxw3um7YpcyjNKSKXCwDisB5Rgnbf97O8JSoT2NAN9hRkK4DMoAoDR+3Hl
WlRSDXd4cvvhmU5wt6oF7hL8RfPVfdvX55uhW1EP/mbXJlAi+8Q3S2XvbZoP7M49
ZsksUGjHIVhoxv83Csz6RZu1wh+EnT97wBX9Q+jGuOJgY56RJycvMcWFSgWaGsgR
Dvpe7LCP2XU9DWVT/tsxbln4o8c6qLN/Aq/53SNfB4z0Wjmzz/hLIHaut3ubiDXh
veWu/s1MwU7bUTMP5GqIP5e/w9ChTUR8Ui/UVco31gZlv1iX1Q7gqVxZLq22RT1u
odsqAXw+kmay5bKJqFZD0A==
=4+7b
-----END PGP MESSAGE-----

**Pull request merged**